### PR TITLE
fix outdated manifest version & use project.version instead

### PR DESCRIPTION
**Pull request by z7658329**

### CODE REVIEW

File: hugegraph-api/pom.xml [hunk 1]
```diff
@@ -186,7 +186,7 @@
                         </manifest>
                         <manifestEntries>
                             <!-- TODO: update it -->
-                            <Implementation-Version>0.69.0.0</Implementation-Version>
+                            <Implementation-Version>${project.version}</Implementation-Version>
```
Comment (imbajin) [inline hugegraph-api/pom.xml:189]: seems we should keep it now, it's APIversion

File: hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java [hunk 2]
```diff
@@ -122,11 +122,11 @@ public final class ApiVersion {
...
      * TODO: what shall we set for this version? (consider the basic compatibility)
      */
-    public static final Version VERSION = Version.of(ApiVersion.class, "0.69");
+    public static final Version VERSION = Version.of(ApiVersion.class, "1.0.0");
 
     public static void check() {
         // Check version of hugegraph-core. Firstly do check from version 0.3
         // TODO: what shall we set for this version? (consider the basic compatibility)
-        VersionUtil.check(CoreVersion.VERSION, "0.13", "0.14", CoreVersion.NAME);
+        VersionUtil.check(CoreVersion.VERSION, "1.0.0", "1.0.1", CoreVersion.NAME);
```
Comment (imbajin) [inline hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java:130]: too strict

File: hugegraph-core/pom.xml [hunk 3]
```diff
@@ -325,7 +325,7 @@
                             </addDefaultSpecificationEntries>
                         </manifest>
                         <manifestEntries>
-                            <Implementation-Version>0.13.0.0</Implementation-Version>
+                            <Implementation-Version>${project.version}</Implementation-Version>
```
Comment (imbajin) [inline hugegraph-core/pom.xml:328]: it's right here

File: hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java [hunk 4]
```diff
@@ -41,6 +41,6 @@ public class CoreVersion {
     public static void check() {
         // Check version of hugegraph-common
         // TODO: why shall we check it? Update it if need
-        VersionUtil.check(CommonVersion.VERSION, "2.1.0", "2.2.0", CommonVersion.NAME);
+        VersionUtil.check(CommonVersion.VERSION, "1.0.0", "1.0.1", CommonVersion.NAME);
```
Comment (imbajin) [inline hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java:44]: maybe 1.0~1.2 first

File: hugegraph-api/pom.xml (hunk 1 above)
Comment (z7658329) [reply hugegraph-api/pom.xml:189]: @javeme  if we should keep all version to 1.0.0 ?

File: hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java (hunk 4 above)
Comment (z7658329) [reply hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java:44]: > 

i will fix it, but ApiVersion should discuss

File: hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java (hunk 2 above)
Comment (z7658329) [reply hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java:130]: keep 1.0~1.2 first ? @imbajin

File: hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java (hunk 4 above)
Comment (javeme) [reply hugegraph-core/src/main/java/org/apache/hugegraph/version/CoreVersion.java:44]: the right version is inclusive, `[1.0, 1.1)` is ok

File: hugegraph-api/pom.xml (hunk 1 above)
Comment (javeme) [reply hugegraph-api/pom.xml:189]: I think we need to keep the api history

File: hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java [hunk 5]
```diff
@@ -122,11 +122,11 @@ public final class ApiVersion {
      * The second parameter of Version.of() is for IDE running without JAR
      * TODO: what shall we set for this version? (consider the basic compatibility)
      */
-    public static final Version VERSION = Version.of(ApiVersion.class, "0.69");
```
Comment (imbajin) [inline hugegraph-api/src/main/java/org/apache/hugegraph/version/ApiVersion.java:125]: here？ shoud keep origin

**Review by imbajin — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**